### DISCUSSION TITLE
Any way of resize the box?

### DISCUSSION THREAD

**Discussion by Poiio78** [dialogic-godot/dialogic#1711] [Q&A]
Hi! 
I'm trying to figure out how I can make the dialog box smaller, since I don't want it to take up the entire screen. 
For example, I want to put it at the bottom of the screen and not make it so big. How can I do it?

**Comment by Jowan-Spooner** [comment]
Hey @Poiio78, 
it's a bit hard to help you because of a lack of information:
- what version do you use?
- what have you already tried?

In version 1.x there is the THEME EDITOR that has a box size setting.

In version 2.x there is the the STYLES EDITOR. However what settings it has depends on the layout/preset you are using.

Maybe this helps. If not feel free to provide more detailed information on your project.

**Reply by XGuyonX** [reply]
Using V2  Visual Novel layout
In Styles I would also like to make the box smaller in the Y axis, The default 110y is the smallest it can get.
If that can happen the down arrow in the display text box would need to be scaled so that text would fit better.

Also: I would also like to off set the box on the X axis so that it is not centered please

**Comment by GoldExperience** [comment]
another way is copy  the default layout, modify it
then change the style
![image](https://github.com/coppolaemilio/dialogic/assets/22605164/124d8f06-e1a4-4833-b793-0ff35bd51a4e)
![image](https://github.com/coppolaemilio/dialogic/assets/22605164/7abfda94-299e-4ddf-befe-278121fae7c3)

**Comment by asheden** [comment]
>2 year old issue
>requires changing basic settings of the project
>first helper has 0 text interpretation skills
>second helper says to do something i already tried and couldn't find any settings, but at least there's a guide
>open the guide
>404
>nobody else replied

support for this add-on is top notch!
